### ISSUE TITLE
Fix links to nigthly and latest docs

### ISSUE THREAD

**Issue by goosemania** [pulp/pulpproject.org#287]
https://docs.pulpproject.org/en/nightly/ point to Pulp 2 nightlies.
https://docs.pulpproject.org/en/latest/ point to Pulp 3.0.0.

**Comment by melcorr** [comment]
Hey @goosemania 
I can do this today

**Comment by goosemania** [comment]
@melcorr , thanks!
FWIW, we have master/nightly/ for Pulp 3 nightly docs.
Maybe we need to revisit what we have and cleanup or at least make links more clear and consistent. It's just the couple I noticed today.

**Comment by melcorr** [comment]
Hey @goosemania 
Sorry - I think I misunderstood what you wanted done here. 
I understood this issue to mean that these links were incorrectly used in the Documentation section of the website. 
When I go through the links, I don't find what you mentioned: 

latest for Pulp 3 links to https://docs.pulpproject.org/
and nightly links to https://docs.pulpproject.org/en/master/nightly/ - this is what you mention above. 

I don't find https://docs.pulpproject.org/en/nightly/  or https://docs.pulpproject.org/en/latest/ - but maybe they're right in front of me. Could you ELI5 this for me?

**Comment by goosemania** [comment]
@bmbouter , @dkliban , do you know/remember where those URL are configured? somewhere on the docs server?

/me keeps forgetting that it's not readthedocs pages :)

**Comment by melcorr** [comment]
https://pulp.plan.io/issues/6723 

Looks like the same thing?

**Comment by goosemania** [comment]
yeah, looks similar, let's close this one.
